modules/codegen/src/main/scala/com/twilio/guardrail/ProtocolGenerator.scala

```diff
@@ -246,7 +246,7 @@ object ProtocolGenerator {
           val isRequired = requiredFields.contains(name)
           SwaggerUtil.propMeta[L, F](prop).flatMap(transformProperty(clsName, needCamelSnakeConversion, concreteTypes)(name, prop, _, isRequired))
       })
-      defn <- renderDTOClass(clsName, params, parents)
+      defn <- renderDTOClass(clsName, params, parents) //check for discriminator, see issue222
```

Comment:
Unnecessary comment -- The test infrastructure prevents regression, trying to work around the comments is undesirable.

Reply:
the idea was to add some messages for future generations trying to understand the code